IPython/core/excolors.py

```diff
@@ -126,6 +127,7 @@ def exception_colors():
         line = C.Red,
         caret = C.Normal,
         Normal = C.Normal,
+        ExecutingNode = 'bg:#005f00'
```

Comment:
```suggestion
        ExecutingNode = 'bg:#005f00',
```

Keep diffs minimal :)